resources/qml/ActionPanel/PrintJobInformation.qml

```diff
@@ -90,14 +90,31 @@ Column
             property var printMaterialCosts: PrintInformation.materialCosts
             property var printMaterialNames: PrintInformation.materialNames
 
+            function putBreaksIfLong(item)
+            {
+                var result = ""
+
+                const max_length = 13  // No longer than 13 -> 'WWWWWWWWWWWWWW' doesn't fit.
```

Comment:
Would it be a good idea to better document the magic number, perhaps make it a constant of sorts defined at the top of the file?

Reply:
Probably, if we decide that it should remain HTML (see below).